roles/postgresql/tasks/postgresql_yum7.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: yum | Initialize postgresql
+  become: yes
+  yum:
+    name: "{{ item }}"
+    state: installed
+    update_cache: yes
+  with_items:
+    - postgresql-server
+    - postgresql-contrib
+    - postgresql-devel
+  tags: [db, postgresql]
+
+- name: yum | Initialize Postgresql
+  become: yes
+  command: postgresql-setup initdb
```

Comment:
See another comment: https://github.com/StackStorm/ansible-st2/pull/78/files#r95995753 about executing an action only when another command is changed